lib/extract/netcdf/.gitignore

```diff
@@ -1,2 +1,2 @@
 extract_netcdf_base.f90
-
+read_kernel_data_mod.f90
```

Comment:
Please add a newline to the end of this file.

Reply:
Done.